actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -360,6 +360,15 @@ def escape_once(html)
         ERB::Util.html_escape_once(html)
       end
 
+      # Transforms a Hash of attributes into HTML Attributes, ready to be
+      # interpolated into ERB.
+      #
+      #   <input <%= escape_attributes(type: :text, aria: { label: "Search" }) %> >
+      #   # => <input type="text" aria-label="Search">
+      def escape_attributes(**attributes)
```

Comment:
I'm not totally sold on this method name, and considered what a good alias would be (I'm thinking of [json_escape and j](https://edgeapi.rubyonrails.org/classes/ERB/Util.html#method-c-json_escape) and [html_escape and h](https://edgeapi.rubyonrails.org/classes/ERB/Util.html#method-c-html_escape).

Reply:
Is `html_attributes` an improvement? Is `attrs` too vague?

Reply:
Is overloading the `html_escape`/`h` helpers viable? They look like they accept String arguments now. We could update the interface to accept a Hash. Is that an improvement over introducing a new method?